src/mono/nuget/Microsoft.NET.Runtime.MonoAOTCompiler.Workload/WorkloadManifest.targets

```diff
@@ -0,0 +1,3 @@
+<Project>
+    <Import Project="Sdk.props" Sdk="Microsoft.NET.Runtime.MonoAOTCompiler.Task" />
```

Comment:
Should this use the `RunAOTCompilation` flag? In the absence of this, wouldn't this get imported in every .NET Core project by default

Reply:
What are the MSBuild properties we need to use for this?

If Blazor is using `$(RunAOTCompilation)`, we should use that on iOS, Android, etc.? Right now we use `$(AotAssemblies)`, but it probably needs to change. What property would enable LLVM?

Reply:
It would be nice if we could all adopt `$(RunAOTCompilation)` as the flag to enable AOT.  

@akoeplinger is there anything with respect to enable/disable LLVM?  It's the default, right?

Reply:
Does this actually need a workload manifest? Isn't this going to be a pack that iOS/Android/etc bring in as part of their AOT workloads, rather than a workload itself?

Reply:
No.  I got stuck thinking workloads and packs had direct relationships.

Reply:
@mhutch you might be interested in: https://github.com/dotnet/sdk/issues/16700

It seems if Blazor WebAssembly & Android try to declare the same pack, we hit an issue. So I think the right idea is to make a manifest that both extend?